2-projects/1-todo-list/script.js

```diff
@@ -13,13 +43,33 @@ let todos = [
 populateTodoList(todos);
 
 // This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
+let addBtn = document.querySelector(".btn");
+addBtn.addEventListener("click", addNewTodo);
+
 function addNewTodo(event) {
-  // The code below prevents the page from refreshing when we click the 'Add Todo' button.
-  event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  event.preventDefault();
+  let input = document.getElementById("todoInput");
+
+  //todos.push({ task: input.value, completed: false });
+  populateTodoList([{ task: input.value, completed: false }]);
+  input.value = "";
 }
 
 // Advanced challenge: Write a fucntion that checks the todos in the todo list and deletes the completed ones (we can check which ones are completed by seeing if they have the line-through styling applied or not).
-function deleteAllCompletedTodos() {
+
+let removeBtn = document.querySelector("#remove-all-completed");
+removeBtn.addEventListener("click", deleteAllCompletedTodos);
+
+function deleteAllCompletedTodos(event) {
   // Write your code here...
+  event.preventDefault();
+
+  let li = document.querySelectorAll("li");
+  let list = document.getElementById("todo-list");
+  li.forEach((task) => {
+    if (task.style.textDecoration === "line-through") {
```

Comment:
This works! You could also apply the styling by adding a class, and check for the class on each `li`